Extension/ASPNET-MVC-Extension/Overview.md

```diff
@@ -23,15 +23,15 @@ The Syncfusion provides the following extension supports in Visual Studio:
 
 **No project selected in Visual Studio**
 
-![Syncfusion Menu when No project selected in Visual Studio](Overview-images/Syncfusion_Menu_OverView1.png)
+![Syncfusion Menu when No project selected in Visual Studio with aspnet.mvc extension](overview_images/aspnet-mvc-extension-menu.png)
```

Comment:
1. Don't change the existing Image ALT Text [Seems Image ALT text character count is too high, so] revert your changes in this PR.

Reply:
changes reverted